specify_pose/proposal.md

```diff
@@ -104,6 +115,20 @@ or use a newline (possibly with hanging indents) if they do not fit on one line.
 To help inform this proposal, the authors conducted a brief survey. See the
 [Survey](#survey) section below for more information.
 
+Regarding other rotation formats, there have been several requests to support
+quaternions as part of the URDF specification (see
+[ros/urdfdom#13](https://github.com/ros/urdfdom/issues/13),
+[ros/urdfdom#123](https://github.com/ros/urdfdom/puul/123),
+[ros/urdfdom_headers#51](https://github.com/ros/urdfdom_headers/pull/51)).
+This is relevant to SDFormat because URDF is a similar specification that uses
+the same Euler angle axis convention to express rotation.
```

Comment:
nit: Since angle-axis is its own representation, can we drop the "axis"?

Reply:
https://github.com/ignitionrobotics/sdf_tutorials/pull/57/commits/335156b5264da5e5b7d41f356862166036920bd1